test/ci/buildkite/ngtf-cpu_ubuntu-bin-build.yaml

```diff
@@ -24,6 +24,8 @@
   
   - command: |
       source /localdisk/buildkite/artifacts/$BUILDKITE_BUILD_ID/venv/bin/activate 
+      pip install -U /localdisk/buildkite-agent/prebuilt_tensorflow_1_15_0/artifacts/tensorflow/tensorflow-1.15.0-cp36-cp36m-linux_x86_64.whl
```

Comment:
Not sure why we need to reinstall tf.

This should have installed
`python3 build_ngtf.py --use_prebuilt_tensorflow  --artifacts /local/ .... `